### PR TITLE
QQ `list` command  support

### PR DESCRIPTION
**Pull request by Aimerny**
# Example
QQ_1722052558360

# Fix
Added cleanup logic for closed connections. Fixed the bug that could not send messages from velocity to QQ
QQ_1722084176754

# BTW
picture message segment could be parsed to text.
21465ba868f555a89ce636f725a2fb24

### CODE REVIEW

File: src/main/java/com/zhanganzhi/chathub/platforms/qq/protocol/QQWsServer.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import com.alibaba.fastjson2.JSON;
 import com.alibaba.fastjson2.JSONReader;
 import com.zhanganzhi.chathub.platforms.qq.dto.QQEvent;
+import lombok.Getter;
```
Comment (Aimerny) [inline src/main/java/com/zhanganzhi/chathub/platforms/qq/protocol/QQWsServer.java:6]:
```suggestion
```

File: src/main/java/com/zhanganzhi/chathub/platforms/qq/protocol/QQWsServer.java [hunk 2]
```diff
@@ -20,6 +21,7 @@ public class QQWsServer extends WebSocketServer {
     private final Queue<QQEvent> qqEventDeque;
 
     @Setter
+    @Getter
```
Comment (Aimerny) [inline src/main/java/com/zhanganzhi/chathub/platforms/qq/protocol/QQWsServer.java:24]:
```suggestion
```

**Comment by AnzhiZhang** [comment]
需要格式化一下

**Comment by Aimerny** [comment]
> 需要格式化一下

done

**Pull request merged**